routes/articleRoutes.js

```diff
@@ -0,0 +1,150 @@
+import express from "express";
+import { PrismaClient } from "@prisma/client";
+
+const router = express.Router();
+const prisma = new PrismaClient();
+
+// [POST] 게시글 등록
+router.post("/", async (req, res) => {
+    try {
+        const { title, content } = req.body;
+
+        if (!title || !content) {
+            return res.status(400).json({ message: "제목과 내용을 모두 입력해주세요." });
+        }
+
+        const newArticle = await prisma.article.create({
+            data: { title, content },
+        });
+
+        return res.status(201).json({
+            message: "게시글이 성공적으로 등록되었습니다.",
+            article: newArticle,
+        });
+    } catch (err) {
+        console.error("게시글 등록 오류:", err);
+        return res.status(500).json({ message: "게시글 등록 실패", error: err.message });
+    }
+});
+
+// [GET] 게시글 상세 조회
+router.get("/:id", async(req, res) => {
+    try {
+        const articleId = parseInt(req.params.id, 10);
+
+        if (isNaN(articleId)) {
+            return res.status(400).json({ message: "유효하지 않은 ID입니다." });
+        }
+        
+        const article = await prisma.article.findUnique({
+            where: { id: articleId },
+            select: { id: true, title: true, content: true, createdAt: true },
+        });
+
+        if (!article) {
+            return res.status(404).json({ message: "게시글을 찾을 수 없습니다."});
+        }
+
+        return res.status(200).json(article);
+    } catch (err) {
+        console.error("게시글 조회 오류", err);
+        return res.status(500).json({ message: "게시글 조회 실패", error: err.message });
+    }
+});
+
+// [PATCH] 게시글 수정
+router.patch("/:id", async(req, res) => {
+    try {
+        const articleId = parseInt(req.params.id, 10);
+        const {title, content} = req.body;
+
+        if (isNaN(articleId)) {
+            return res.status(404).json({ message: "유효하지 않은 ID입니다." });
```

Comment:
요청 파라미터가 유효하지 않다. 라는 의미에는 404보다는 400이 더 어울립니다.
```suggestion
            return res.status(400).json({ message: "유효하지 않은 ID입니다." });
```